tests/Admin/AdminTest.php

```diff
@@ -1294,35 +1294,56 @@ public function testDeterminedPerPageValue(): void
     public function testIsGranted(): void
     {
         $admin = new PostAdmin('sonata.post.admin.post', 'Acme\NewsBundle\Entity\Post', 'Sonata\NewsBundle\Controller\PostAdminController');
+        $modelManager = $this->createMock(ModelManagerInterface::class);
+        $modelManager
+            ->method('getNormalizedIdentifier')
+            ->willReturnCallback(static function (?object $entity = null): ?string {
```

Comment:
`getNormalizedIdentifier` is supposed to return a `string`.

Reply:
Yes, but that isn't actually happening. See https://github.com/sonata-project/SonataAdminBundle/pull/6005#issuecomment-611712319.